app/models/game.rb

```diff
@@ -0,0 +1,9 @@
+class Game < ActiveRecord::Base
+  has_many :memberships
+  has_many :users, through: :memberships
+
+  def <<(user)
```

Comment:
the syntactic sugar factor seems pretty weak here. just lets you avoid typing a `.`? I personally think the `<<` name just obscures the functionality of the method. If the class actually implemented the Enumerable interface, and could be treated like an array in other ways, I might feel differently.